xbmc/FileItem.cpp

```diff
@@ -3741,14 +3741,14 @@ bool CFileItem::LoadDetails()
     return true;
   }
 
-  if (m_bIsFolder && URIUtils::IsPVRRecordingFileOrFolder(GetPath()))
+  if (URIUtils::IsPVRRecordingFileOrFolder(GetPath()))
```

Comment:
I think this is likely a getdynpath here. I.e. a plugin:// item could resolve to a Pvr recording right? In such cases behaviour would differ if playing the plugin item or the Pvr recording directory.

Reply:
Nope. I need to check the pvr-URL, which is the unchangable identity of the item.

> plugin:// item could resolve to a Pvr recording right

Those cases currently will not work as by design. Dyn path is just a single de-reference, what you are referring to would need a chain of path: plugin -> pvr -> file